src/devices/honyar.ts

```diff
@@ -54,30 +70,27 @@ const definitions: Definition[] = [
         model: 'U86Z223A10-ZJU01(GD)',
         vendor: 'Honyar',
         description: 'Smart power socket 10A with USB (with power monitoring)',
-        fromZigbee: [fz.on_off, fz.electrical_measurement, fz.metering],
+        fromZigbee: [fzLocal.HY0157_on_off, fz.electrical_measurement, fz.metering, fz.ignore_basic_report],
```

Comment:
You can add `meta: {multiEndpoint: true, multiEndpointSkip: ['energy', 'current', 'voltage', 'power']},` instead of adding `HY0157_on_off`